samples/ingestion/ingestion-client/versions.nugets.props

```diff
@@ -6,6 +6,9 @@
 
     <MSTestTestAdapterPackageVersion>[3.4.3]</MSTestTestAdapterPackageVersion>
     <MSTestTestFrameworkPackageVersion>[3.4.3]</MSTestTestFrameworkPackageVersion>
+    <MSTestSdk>[17.10.0]</MSTestSdk>
```

Comment:
please follow convention of other packages

Reply:
Moved to 3rd party packages